ocaml/testsuite/tests/typing-missing-cmi-2/test.ml

```diff
@@ -13,5 +13,3 @@ module = "baz.ml"
 ocamlc_byte_exit_status = "2"
 ****** check-ocamlc.byte-output
 *)
-
-(* CR layouts v2.9: error message here is unreviewed *)
```

Comment:
`ocaml/testsuite/tests/typing-missing-cmi-2/test.compilers.reference`

This one is good, and I don't have a real issue with it. A nit:

```
Error: This expression has type 'a Foo.t
       but an expression was expected of type ('b : '_representable_layout_1)
       [...]
       But the layout of 'a Foo.t must be a sublayout of '_representable_layout_1, because
         it's the type of a variable bound by a `let`.
```

The first two lines read well. This doesn't read as well: `But the layout of 'a Foo.t must be a sublayout of '_representable_layout_1`.

This might read better: `But the layout of 'a Foo.t must be representable.`

This is just a nit, I think this message is fine.

Reply:
suggested change implemented